src/common/rand/rand.c

```diff
@@ -107,6 +122,7 @@ void OQS_randombytes_system(uint8_t *random_array, size_t bytes_to_read) {
 	CryptReleaseContext(hCryptProv, 0);
 }
 #endif
+#endif
```

Comment:
Suggest adding a comment as to which #define(s) gets closed here

Reply:
done